lib/notesLinks.ml

```diff
@@ -59,66 +59,68 @@ let misc_notes_link s i =
     else
       let j =
         let rec loop j =
-          if j = slen then j
-          else if j < slen - 2 && s.[j] = ']' && s.[j+1] = ']' then j + 2
+          if j = slen then i
+          else if j <= slen - 2 && s.[j] = ']' && s.[j+1] = ']' then j + 2
           else loop (j + 1)
         in
         loop (i + 2)
       in
-      let b = String.sub s (i + 2) (j - i - 4) in
-      let (spe, b) =
-        try
-          let i = String.index b ':' in
-          Some (String.sub b 0 i),
-          String.sub b (i + 1) (String.length b - i - 1)
-        with Not_found -> None, b
-      in
-      let (b, text) =
-        try
-          let i = String.rindex b ';' in
-          String.sub b 0 i,
-          Some (String.sub b (i + 1) (String.length b - i - 1))
-        with Not_found -> b, None
-      in
-      if spe = Some "w" then
-        let (wiz, name) =
-          match try Some (String.index b '/') with Not_found -> None with
-            Some i ->
-              String.sub b 0 i, String.sub b (i + 1) (String.length b - i - 1)
-          | None -> b, ""
+      if j > i + 4 then
```

Comment:
I think it should be `j > i + 4 + 2`.

Reply:
il me semble que si on a [[]] (cas minimum) alors j = i+4

Reply:
Hum... possible, j'étais parti sur un `j` qui pouvait avoir n'importe quelle valeur et qui donc produirait une erreur pour `String.sub s (i + 2) (j - i - 4)`

Reply:
Pour sortir de la boucle, il faut deux ]]. Si on a pas les deux ]] j=i!

Reply:
Au temps pour moi !